{
	"Settings": 
	[
		{
			"Name": "OrchestratorQueueName",
			"Value": "ProcessABCQueue",
			"Description": ""
		},
		{
			"Name": "OrchestratorQueueFolder",
			"Value": "",
			"Description": ""
		},
		{
			"Name": "logF_BusinessProcessName",
			"Value": "Framework",
			"Description": ""
		}
	],
	"Constants":
	[
		{
			"Name": "MaxRetryCount",
			"Value": "0",
			"Description": ""
		},
		{
			"Name": "MaxConsecutiveSystemExceptions",
			"Value": "0",
			"Description": ""
		},
		{
			"Name": "ExScreenshotsFolderPath",
			"Value": "Exceptions_Screenshots",
			"Description": ""
		},
		{
			"Name": "LogMessage_GetTransactionData",
			"Value": "Processing Transaction Number:",
			"Description": ""
		},
		{
			"Name": "LogMessage_GetTransactionDataError",
			"Value": "Error getting transaction data for Transaction Number:",
			"Description": ""
		},
		{
			"Name": "LogMessage_Success",
			"Value": "Transaction Successful",
			"Description": ""
		},	
		{
			"Name": "LogMessage_BusinessRuleException",
			"Value": "Business rule exception.",
			"Description": ""
		},
		{
			"Name": "LogMessage_ApplicationException",
			"Value": "System exception.",
			"Description": ""
		},
		{
			"Name": "ExceptionMessage_ConsecutiveErrors",
			"Value": "The maximum number of consecutive system exceptions was reached.",
			"Description": ""
		},
		{
			"Name": "RetryNumberGetTransactionItem",
			"Value": "2",
			"Description": ""
		},
		{
			"Name": "RetryNumberSetTransactionStatus",
			"Value": "2",
			"Description": ""
		},
		{
			"Name": "ShouldMarkJobAsFaulted",
			"Value": "FALSE",
			"Description": ""
		},
		{
			"Name": "WorkItemType",
			"Value": "WI4",
			"Description": ""
		},	
		{
			"Name": "ReportYear",
			"Value": "now.Year - 1",
			"Description": ""
		} 	
	],
	"Assets":
	[
		{
			"Name": "",
			"Value": "",
			"Description": ""
		}
	]
}